analyses/cell-type-ewings/exploratory_analysis/06-tumor-cell-state-assignment.Rmd

```diff
@@ -0,0 +1,606 @@
+The chondrocytes also express high markers of mesenchymal-like genes and the literature talks about tumor cells with low `EWS-FLI1` expression being "MSC-like". 
+I think there's a possibility that these are tumor cells, so let's relabel them. 
+
+```{r}
+# create a new tumor classification that takes all tumor clusters plus the clusters with EWS-FLI1 low expression 
+cluster_classification_df <- cluster_classification_df |> 
+  dplyr::mutate(
+    tumor_classification = dplyr::case_when(
+      cluster_classification == "tumor" ~ "tumor",
+      library_id == "SCPCL000822" & cluster_classification == "chondrocyte" ~ "tumor",
+      library_id == "SCPCL000824" & cluster_classification == "mesenchymal-like cell" ~ "tumor",
+      .default = cluster_classification
+    )
+  )
+```
+
+
+## Classify tumor cell states 
+
+Now that we have identified a subset of cells that are tumor cells, we can attempt to group them into various cell states. 
+The main thing that we are interested in is grouping cells into levels of `EWS-FLI1` expression. 
+Let's start by looking at the expression of the gene sets across all tumor cells. 
+
+To do this, we will make a density plot that looks at gene set expression by cluster using the cluster assignments calculated using all cells. 
+
+```{r}
+tumor_cells_df <- cluster_classification_df |> 
+  dplyr::filter(tumor_classification == "tumor")
+```
+
+```{r, fig.height=10}
+# density plot showing expression of each gene set by cluster
+library_ids |> 
+  purrr::map(\(id){
+    
+    density_df <- tumor_cells_df |> 
+      dplyr::filter(library_id == id)
+    
+    marker_gene_cols |>
+    purrr::map(\(column){
+      plot_density(
+        density_df,
+        column,
+        "cluster"
+      ) 
+    }) |>
+    patchwork::wrap_plots(ncol = 1) +
+      patchwork::plot_annotation(id)
+    
+  })
+```
+
+A few notes from this: 
+
+- It doesn't look like very many cells express proliferative markers, so perhaps that gene set will not be helpful for these libraries? 
+- There are some very obvious `EWS-FLI1` low groups of cells. 
+- The `EWS-high` and `aynaud-EWS-high` show clusters that have high expression of `EWS-FLI1` targets, but the `aynaud-EWS-high` appears to have some more separation across clusters. 
+That tells me that gene signature might be more helpful in classifying cells across the `EWS-FLI1` continuum. 
+
+### Sub cluster tumor cells 
+
+We would expect that tumor cells that are in similar cell states would cluster together. 
+So here I am going to take just the tumor cells and re-assign clusters. 
+I'm using the same parameters we use for the whole sample, leiden, modularity, resolution of 0.5, and nearest neighbors of 20. 
+I think if we really wanted to be stringent we would evaluate clusters here and find the best parameters to use, but I think starting with these is good. 
```

Comment:
I don't think there are "perfect" answers here since clustering is its own special can of worms. But, I do want to say that I wonder what the difference would be between using the same parameters on subsetted data, vs using the full dataset with a higher resolution. I agree this is a decent enough place to start though, since it's anyone's guess how to subset clusters anyways!